server/schedule/operator/kind.go

```diff
@@ -42,32 +42,36 @@ const (
 	OpRegion
 	// Include leader transfer.
 	OpLeader
+	// Include witness leader transfer.
+	OpWitnessLeader
```

Comment:
where is it used?

Reply:
Fixed, sorry, the change was in `git stashed`...